backend/alembic/versions/0021_fav_sibling.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 0021_fav_sibling
+Revises: 0020_created_and_updated
+Create Date: 2024-06-27 18:04:55.707623
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0021_fav_sibling"
+down_revision = "0020_created_and_updated"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("roms", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "fav_sibling", sa.Boolean(), nullable=True, server_default=sa.false()
+            )
+        )
```

Comment:
This needs to relate to the user somehow, we can't have it be global for every rom. You can probably rename and extend RomNote to something more generic, that includes fields like this one.